protomap/proto.go

```diff
@@ -920,6 +939,12 @@ func hasValuePathPrefix(opts []UnmapOpt) (*gpb.Path, error) {
 // type of payload is provided for the message. The second, boolean, return argument specifies whether
 // the message provided was a known wrapper type.
 func makeWrapper(msg protoreflect.Message, fd protoreflect.FieldDescriptor, val interface{}) (protoreflect.Message, bool, error) {
```

Comment:
It looks like this function currently doesn't handle `wpb.IntValue` and `wpb.BoolValue`. Add a TODO?

Reply:
Done.